include/boost/type_traits/detail/config.hpp

```diff
@@ -48,7 +48,7 @@
 // when we want to test __stdcall etc function types with is_function etc
 // (Note, does not work with Borland, even though it does support __stdcall etc):
 //
-#if defined(_MSC_EXTENSIONS) && !defined(__BORLANDC__)
+#if defined(_MSC_EXTENSIONS) && !defined(BOOST_BORLANDC)
```

Comment:
Has this been tested?  Does the clang based compiler support msvc calling conventions?  Just double checking....

Reply:
The tests all pass. Clang says it supports the msvc calling conventions.